13/umbraco-cms/fundamentals/setup/upgrading/version-specific/upgrade-from-8-to-latest.md

```diff
@@ -7,9 +7,9 @@ description: >-
 # Upgrade from Umbraco 8 to the latest version
```

Comment:
Should this article maybe be renamed from _Upgrade from Umbraco 8 to the latest version_  To _Upgrade from Umbraco 8 to Umbraco 10_ As that is what you technically need to do 🤔

Reply:
Yes, I guess it makes sense.